common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1354,6 +1355,15 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("600s")
 
+  val WORKER_EXCLUDE_SLAVE_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.excluded.slave.enabled")
```

Comment:
better to rename to celeborn.client.blacklistSlave.enabled

Reply:
Done